multinode_integration_tests/src/neighborhood_constructor.rs

```diff
@@ -98,7 +98,9 @@ fn make_mock_node_map(
         .map(|node| (node.main_public_key().clone(), node))
         .collect::<HashMap<PublicKey, MASQMockNode>>();
     additional_keys_to_mock.iter().for_each(|key| {
-        let mock_node = cluster.start_mock_node_with_public_key(vec![10000], key);
```

Comment:
Does this force every mock Node to start as configurable?
That seems...oppressive. At least could there be a function or method that does it behind the scenes so you don't have to see it?